src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -6,19 +6,104 @@
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_CAPACITY = 16;
+    static final float DEFAULT_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int size = 0;
+
+    public MyHashMap() {
+        table = (Node<K, V>[]) new Node[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size > table.length * DEFAULT_FACTOR) {
+            resize();
+        }
+        putVal(key, value, table, true);
     }
 
     @Override
     public V getValue(K key) {
+        if (table != null) {
+            if (table.length > 0) {
+                if (table[getIndex(key, table.length)] != null) {
+                    Node<K, V> node = table[getIndex(key, table.length)];
+                    if (node != null) {
+                        do {
+                            if (compareKey(node.key, key)) {
+                                return node.value;
+                            }
+                            node = node.next;
+                        } while (node != null);
+                    }
+                }
+            }
+        }
         return null;
     }
 
     @Override
     public int getSize() {
-        return 0;
+        return size;
+    }
+
+    private static final int hash(Object key) {
```

Comment:
```suggestion
    private static final int hash(Object key) {
```
чому static? чому final? ти сама написала цей метод?

Reply:
Я все пишу сама! Основывалась на исходник HashMap. Delete static final